jena-core/src/main/java/org/apache/jena/vocabulary/XSD.java

```diff
@@ -178,6 +180,36 @@ public class XSD {
     /** Resource URI for xsd:gMonthDay */
     public static Resource gMonthDay;
 
+    /** Property URI for xsd:length */
```

Comment:
Is "Property" the right word here? These are constraining facets, which must be an IRI, so maybe Property is forced.

Reply:
I meant `org.apache.jena.rdf.model.Property`, it is used as a predicate in facet restrictions